packages/wrangler/src/core/define-command.ts

```diff
@@ -101,13 +101,20 @@ export type CommandDefinition<
 		 * @default true
 		 */
 		printBanner?: boolean;
+
+		/**
+		 * By default, wrangler will print warnings about wrangler.toml configuration.
+		 * Set this value to `false` to skip printing these warnings.
+		 */
+		printConfigWarnings?: boolean;
```

Comment:
I was rebasing the changes on #6756 over to this PR and notice the same changes doesn't work here. The reason was that `defineCommand` will read the config for you and there was no way to disable the warning.  You can find the full changes that enable this option in https://github.com/cloudflare/workers-sdk/pull/7150/commits/ec6d4cd46074138990400971432afa90566229f6

cc: @penalosa @emily-shen